src/api/types.ts

```diff
@@ -311,3 +311,9 @@ export interface ReferencedForm {
   formName: string;
   alias: string;
 }
+
+export type TobsGroupCounter = {
```

Comment:
This naming doesn't look intuitive enough especially with the `T` can rename it to `RepeatGroupCounter` or `ObsGroupCounter` or something else

Reply:
This is resolved.